go.mod

```diff
@@ -5,13 +5,18 @@ go 1.22.7
 toolchain go1.23.3
 
 require (
-	github.com/aws/aws-sdk-go v1.55.5
-	github.com/hashicorp/aws-sdk-go-base v1.1.0
+	github.com/aws/aws-sdk-go-v2 v1.32.7
```

Comment:
I confirm that this upgrade resolves https://github.com/idealo/terraform-provider-controltower/issues/211